CharlieBackend.Api/Controllers/AccountsController.cs

```diff
@@ -104,7 +104,7 @@ public async Task<ActionResult> SignIn([FromBody]AuthenticationDto authenticatio
                     return BadRequest();
                 }
 
-                entityId = foundMentor.Id;
+                entityId = foundMentor.Data.Id;
```

Comment:
You're risking to get NullReferenceException here. foundMentor will be Result<MentorDto> (always non-null) but data could be null if mentor wasn't found